ocs_ci/deployment/deployment.py

```diff
@@ -1012,6 +1012,49 @@ def deploy_ocs_via_operator(self, image=None):
 
         # Create Multus Networks
         if config.ENV_DATA.get("is_multus_enabled"):
+            from ocs_ci.deployment.nmstate import NMStateInstaller
+
+            logger.info("Install NMState operator and create an instance")
+            nmstate_obj = NMStateInstaller()
+            nmstate_obj.running_nmstate()
+            logger.info("Configure NodeNetworkConfigurationPolicy on all worker nodes")
+            worker_node_names = get_worker_nodes()
+            for worker_node_name in worker_node_names:
```

Comment:
do we have add node operation running on BM env?  If so, we might think of configure also 4th node argo008.
And we need to then think of invoking those changes for that specific node during add node operation.

Doesn't need to be in this pr implemented for add node even though we use it, but if we are adding node we need to implement this later on I guess.